wyvern/wyvern_request.py

```diff
@@ -77,3 +77,6 @@ def parse_fastapi_request(
             feature_map=FeatureMap(feature_map={}),
             request_id=request_id,
         )
+
+    def extend_feature_map(self, feature_map: FeatureMap) -> None:
+        self.feature_map.feature_map.update(feature_map.feature_map)
```

Comment:
Do we have any duplication check? If not, let's create a ticket for adding one, wdyt?

Reply:
What kind of duplicate use case you're thinking of? is it overriding an existing key by another existing key?

The the self.feature_map.feature_map is already a python dictionary and it should have no duplicates.

Reply:
Is it possible that they define one offline, one real time feautre view and they both have the same name and same feature name?